### ISSUE TITLE
how to remove home directory indicator "~" or "/"

### ISSUE THREAD

**Issue by Showvro** [romkatv/powerlevel10k#1683]
→ ~
 → /src on main  
how can I remove this indicator from p10k

**Comment by romkatv** [comment]
What do you want to see when your current directory is:

- /
- /foo
- /foo/bar
- ~
- ~/foo
- ~foo
- ~foo/bar

?

**Comment by Showvro** [comment]
Can I keep empty or whitespace without an indicator like the inserted image?

Thanks for your valuable time ❤

**Comment by romkatv** [comment]
I done understand your answer. Can you rephrase it like this? "When my prompt is X, I want to see Y." For every X from the list above.

**Comment by Showvro** [comment]
I tried to make it happen but I can't resolve it. Please give me
instructions or suggestions.

**Comment by romkatv** [comment]
I'll help you if you answer my questions.

**Comment by Showvro** [comment]
Ask me whatever do you want to know

**Comment by romkatv** [comment]
Closing because communication is not working.
